sdk/communication/arm-communication/README.md

```diff
@@ -63,7 +63,17 @@ const client = new CommunicationServiceManagementClient(new DefaultAzureCredenti
 // });
 // const client = new CommunicationServiceManagementClient(credential, subscriptionId);
 ```
-
+```javascript
```

Comment:
I get that you add this, but where do I use this resource object that you're declaring here to create my resource with a system assigned identity?

Reply:
to use the resource object that it has been declared here to create a resource with a system-assigned identity, can follow these steps: 
1. create a new instance of the resource
2. set the identity property of the resource to an object that specifies the type of identity you want to use 
3. use the new instance of the resource to create your resource
const result = await communicationServices.beginCreateOrUpdate(resourceGroup, resourceName, resource);

Reply:
Is this information in the readme already?